content/information-distribution/article.md

```diff
@@ -12,22 +12,12 @@ signature coming from its source, and be published on the publication name space
 that has been declared by its source. The article **can** also include optional
 properties, such as the item's description, read duration, NSFW marker, etc..
 
-The signature **must** be generated from one of the source's public signature
-verification keys (using the source's declared signing algorithm), and a string
-containing all of the item's information, and **must** be formatted the same way
-as follows:
-
-```
-title=Lorem ipsum&author=Cicero&date=2006-01-06T15:04-07:00&url=https://lipsum.com/&content=Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-```
-
-In the example string above, the target article is titled "Lorem ipsum", was
-written by "Cicero" on January 1st 2006 and can be found at
-"[https://lipsum.com/](https://lipsum.com/)" with the content "Lorem ipsum dolor
-sit amet, consectetur adipiscing elit.".
+The article JSON object **must** be encapsulated in a [signed matrix
+event](/information-distribution/signature), using one of the source's keys.
```

Comment:
For ultimate clarity, I'd even say "with its signature generated with one of the source's keys".
Also, capital first letter for "Matrix".